### PR TITLE
[/tg/ Balancing] Unfucks the Spray Cleaner

### PR DESCRIPTION
**Pull request by Manly-Dwarf**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: Kawaii-Big-Boss
fix: Spray Cleaner is no longer retarded.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

#7060 brought us "wonderful" changes to the Spray Cleaner. It reduced the volume it could hold to 100, made it so that you only apply two units to a tile, and nerfed spray cleaner probably because he didn't like being spray tanned/slipped with water or lube.

You could still use pest spray for this, c'mon. No need to make the janitor's bottle only good for spray cleaner.

### CODE REVIEW

File: hippiestation/code/modules/reagents/reagent_containers/spray.dm [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/obj/item/reagent_containers/spray/cleaner
+	name = "space cleaner"
+	desc = "BLAM!-brand non-foaming space cleaner!"
+	volume = 250
+	list_reagents = list("cleaner" = 250)
+	amount_per_transfer_from_this = 5
+	stream_amount = 5
```
Comment (JohnGinnane) [inline hippiestation/code/modules/reagents/reagent_containers/spray.dm]: The default value for stream_amount is 10 btw, so if you want to completely revert /tg/'s changes you should set this to 10

**Review by JohnGinnane — DISMISSED** [review]
Code is :ok:

**Comment by JohnGinnane** [comment]
Not 100% if we need headmin for this actually @Carbonhell @Pyko1 ?

**Comment by GuyonBroadway** [comment]
to be fair with a lighter and a 250u spraycleaner bottle of blackpoweder you CAN make a ghetto maxcap

But most people don't as you're giving up a spraycleaner bottle.

**Comment by JohnGinnane** [comment]
I think it's mostly we don't wanted tg balance changes because our playerbase is different

**Comment by Jujumatic** [comment]
namely our playerbase isn't as retarded as tg's, just by far more autistic.

hopefully this is the first step into reverting some more of the stupid ass retarded changes and nerfs the tg menace has done

**Comment by ghost** [comment]
honestly i don't see what the huge issue was
nothing stopping you from carrying a beaker of the solution you want, i mean, that sort of defeats the whole volume nerf since it's easy as fuck to counter, just annoying

File: hippiestation/code/modules/reagents/reagent_containers/spray.dm [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/obj/item/reagent_containers/spray/cleaner
+	name = "space cleaner"
```
Comment (ghost) [inline hippiestation/code/modules/reagents/reagent_containers/spray.dm]: nitpicking here, but this was never modified and doesn't really need to be here

File: hippiestation/code/modules/reagents/reagent_containers/spray.dm [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+	desc = "BLAM!-brand non-foaming space cleaner!"
```
Comment (ghost) [inline hippiestation/code/modules/reagents/reagent_containers/spray.dm]: ditto

**Comment by Manly-Dwarf** [comment]
That's precisely the problem I have with it, Bawb. It's such an unnecessary and stupid change that is hilariously easy to get around. Why even have it to begin with?

**Comment by ghost** [comment]
+1

**Comment by Carbonhell** [comment]
+1 but code needs to be fixed

**Comment by Pyko1** [comment]
I find it really stupid that a spraycan can is actualyl one of the largest containers you can carry around.

If you ask me it should be smaller than a bucket.

**Comment by JohnGinnane** [comment]
@Pyko1 I think the bucket should be bigger to compensate, as it's the one that's disproportionate. Beakers are 50 to 100u, and a bucket is only 70?

**Comment by Pyko1** [comment]
Cold prolly make buckets not being beakers and up the size to 150-200. But still I find silly that a spray bottle can hold 1/4th of a chem tank.

**Comment by praisenarsie** [comment]
dont nerf janitor please

**Comment by JohnGinnane** [comment]
if a head says -1 what happens to the pr

**Comment by Carbonhell** [comment]
Realism is nice but imo it isn't needed in this case, as replenishing a spray every five uses is boring as fuck. I stand with my +1, we gotta wait for tfw

**Comment by ghost** [comment]
it's the same efficieny carbon
all it does is decrease the volume and how much it uses a spray
space cleaner doesn't care about the volume so it doesn't negatively affect it.

the pr made it so it's harder to use cleaners as bombs (i hadn't realised this was a thing) but also harder to use them as weapons

**Comment by JohnGinnane** [comment]
@t-f-w pls))

**Comment by Manly-Dwarf** [comment]
@McBawbaggings I'm pretty sure the main reason it was changed wasn't thanks to Spray Cleaner bombs at all. Based on Damous getting banned for using anti-stuns, I'm not sure if people on /tg/ even know they can do that.

**Comment by earthmoonjune** [comment]
ye +1

**Review by JohnGinnane — DISMISSED** [review]
McBaw is right, I think we should remove the name and desc incase it's changed upstream other than that it's good to go

**Comment by Pyko1** [comment]
removed name and desc, and the PR is approved, someone merge when travis is done with his stuff.

**Review by GuyonBroadway — APPROVED** [review]

**Review by JohnGinnane — APPROVED** [review]

**Comment by GuyonBroadway** [comment]
codes good and has a solid +2

**Pull request merged**